.github/workflows/cd.yml

```diff
@@ -29,10 +29,7 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - python-version: "3.8"
-            lammps-version: "2020.12.24"
-            backend: django
```

Comment:
I think you can remove all the other `backend` occurrences. The difference between django/sqlalchemy was removed in AiiDA v2, it no longer makes sense.

Reply:
Great! thanks, I suspected it just was not sure why it was there in the latest release.